src/Sylius/Component/Order/Model/Adjustment.php

```diff
@@ -105,6 +110,10 @@ public function getAdjustable()
             return $this->orderItem;
         }
 
+        if (null !== $this->orderItemUnit) {
+            return $this->orderItemUnit;
+        }
+
         return null;
```

Comment:
Ok, sadly :-1: After looking at this it would return bool, but we expect not-null object or ``null`` ;)